3rdParty/StormLib/src/FileStream.cpp

```diff
@@ -564,7 +564,7 @@ static void BaseMap_Close(TFileStream * pStream)
         UnmapViewOfFile(pStream->Base.Map.pbFile);
 #endif
 
-#if (defined(PLATFORM_MAC) || defined(PLATFORM_LINUX) || defined(PLATFORM_HAIKU)) && !defined(PLATFORM_AMIGA)
+#if (defined(PLATFORM_MAC) || defined(PLATFORM_LINUX) || defined(PLATFORM_HAIKU)) && !defined(PLATFORM_AMIGA) || !defined(PLATFORM_CTR)

```

Comment:
Probably this has gotten a bit mangled with &&/¦¦, this might fix Windows:
```suggestion
#if (defined(PLATFORM_MAC) || defined(PLATFORM_LINUX) || defined(PLATFORM_HAIKU)) && !defined(PLATFORM_AMIGA) && !defined(PLATFORM_CTR)
```
But i'm not sure you would even need to touch this line as you shoudn't fall under `defined(PLATFORM_MAC) || defined(PLATFORM_LINUX) || defined(PLATFORM_HAIKU)` (and neither should PLATFORM_AMIGA for that matter)